models/staging/stg_contract_data.yml

```diff
@@ -0,0 +1,118 @@
+version: 2
+
+models:
+  - name: stg_contract_data
+    tests:
+      - dbt_utils.recency:
+          datepart: day
+          field: cast(closed_at as timestamp)
+          interval: 2
+          meta:
+            description: "Monitors the freshness of your table over time, as the expected time between data updates."
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - contract_id
+            - contract_data_xdr
+            - last_modified_ledger
+            - ledger_entry_change
```

Comment:
Suggestion: I would use `ledger_key_hash` with `last_modified_ledger` and `ledger_entry_change`

Reply:
Resolved in https://github.com/stellar/stellar-dbt-public/pull/100/commits/e41f9a26bc1044392dc3c92eba23a10f8d995fad